bbtools crash causes pipeline stalling

Possible multi-threading bug on bbtools 39.49 

```
1054.matched_phix.fq, ref=/bbmap/resources/phix174_ill.ref.fa.gz, k=31, hdist=1, stats=2016V-1054.phix.stats.txt]
Version 39.49

Allocating kmer table:  0.013 seconds.
Initial:
Memory: max=8061m, total=8061m, free=8019m, used=42m

Added 487396 kmers; time:       0.047 seconds.
Memory: max=8061m, total=8061m, free=7995m, used=66m

Input is being processed as paired
Started output streams: 0.027 seconds.
Exception in thread "BGZF-InputProducer" java.lang.AssertionError: Not a gzip file: 0, 0
        at stream.bam.BgzfInputStreamMT2.readNextBlock(BgzfInputStreamMT2.java:212)
        at stream.bam.BgzfInputStreamMT2.producerLoop(BgzfInputStreamMT2.java:132)
        at stream.bam.BgzfInputStreamMT2.access$0(BgzfInputStreamMT2.java:124)
        at stream.bam.BgzfInputStreamMT2$1.run(BgzfInputStreamMT2.java:106)
        at java.base/java.lang.Thread.run(Thread.java:1583)
```

--- a/tasks/task_bbtools.wdl
+++ b/tasks/task_bbtools.wdl
@@ -31,6 +31,8 @@ task assembly_prep {
       ref=/bbmap/resources/phix174_ill.ref.fa.gz \
       k=31 \
       hdist=1 \
+      bgzip=f \
+      unbgzip=f \
       stats=~{samplename}.phix.stats.txt
 
     grep Matched ~{samplename}.phix.stats.txt | awk '{print $3}' > PHIX_RATIO

--- a/tasks/task_version.wdl
+++ b/tasks/task_version.wdl
@@ -9,7 +9,7 @@ task version_capture {
   }
 
   command <<<
-    cbird_version="C-BIRD v2.3.0"
+    cbird_version="C-BIRD v2.3.1"
     ~{default='' 'export TZ=' + timezone}
     date +"%Y-%m-%d" > TODAY
     echo "$cbird_version" > CBIRD_VERSION

